opentelemetry-appender-tracing/Cargo.toml

```diff
@@ -13,10 +13,12 @@ rust-version = "1.75.0"
 [dependencies]
 log = { workspace = true, optional = true }
 opentelemetry = { version = "0.27", path = "../opentelemetry", features = ["logs"] }
+opentelemetry_sdk = { version = "0.27", path = "../opentelemetry-sdk", features = ["logs"]  }
```

Comment:
this is not desirable. Log appenders should only be dependent on the opentelemetry crate. Not able to see why this dependency is needed?

Reply:
Oh, I don't think it ended up being necessary.  I've pushed a commit to remove it.